### PR TITLE
LOG-4809: reduce stream file size for functional-benchmarker

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR:
* Modifies the functional-benchmarker to reduce the amount of work being done by the utility and offloads some of it to the receiver

### Links
* https://issues.redhat.com/browse/LOG-4809

cc @syedriko

### CODE REVIEW

File: internal/cmd/functional-benchmarker/runners/cluster/cluster_runner.go [hunk 1]
```diff
@@ -78,8 +78,8 @@ func (r *ClusterRunner) Deploy() {
 			conf = strings.Replace(conf, "/var/log/pods/*/*/*.log", pattern, 1)
 			conf = strings.Replace(conf, "/var/log/pods/**/*.log", pattern, 1)
 		case logging.LogCollectionTypeVector:
-			pattern := `exclude_paths_glob_patterns = ["/var/log/pods/openshift-logging_collector-*/*/*.log"`
-			conf = strings.Replace(conf, `exclude_paths_glob_patterns = ["/var/log/pods/*/collector/*.log"`, pattern, 1)
+			re, _ := regexp.Compile(".*exclude_paths_glob_patterns.*=.*\\[.*\\].*")
```
[the remark this answers is not in the document]
Comment (jcantrill) [reply internal/cmd/functional-benchmarker/runners/cluster/cluster_runner.go:81]: What specifically?

[the remark this answers is not in the document]
Comment (syedriko) [reply internal/cmd/functional-benchmarker/runners/cluster/cluster_runner.go:81]: the regexp.Compile and its result, so it only runs once

**Pull request merged**